Readme.md

```diff
@@ -0,0 +1,55 @@
+# KOB Email Verification Service
+##  Abstract
+ *  KOB Email Verification Services proves the ownership of user over his email. It provides verified digital credential for user’s email which is stored in user’s wallet. User     submit this digital credential, when he provides his email for availing third party service.
+ 
+##  Dependant Project
+ *  This project is dependant on BCGOV's Email Verification Service.
+ 
+##  Motivation
+ *  Current email security solutions depend on various attributes to reduce the chances that a given email (mail) is likely to be a threat.current solutions make it relatively       easy to target corporate organizations with a Business Email Compromise (BEC) attack. To overcome this problem a blockchain based ledger service marks down an outgoing mail.
```

Comment:
- Focus on problems faced under this section and explain the solution part under solution section. 
- You can also talk about something based on this context - Email is one of the most shared digital identities across the internet and it is prone to fraudulent activities.